src/daft-local-plan/src/translate.rs

```diff
@@ -58,9 +58,11 @@ pub fn translate(plan: &LogicalPlanRef) -> DaftResult<LocalPhysicalPlanRef> {
         }
         LogicalPlan::Project(project) => {
             let input = translate(&project.input)?;
+            let resource_request = get_resource_request(&project.projection);
```

Comment:
Is the resource request not derivable directly from the projection?

I.e. instead of defining `resource_request` as a variable on `LocalPhysicalPlan::project`, why not make it a method and run `get_resource_request` on demand?

Reply:
Makse sense!